mod-orders-storage/schemas/details.json

```diff
@@ -21,8 +21,11 @@
           },
           "productIdType": {
             "description": "the type of id",
-            "type": "object",
             "$ref": "../../common/schemas/product_id_type.json"
+          },
+          "qualifier": {
+            "description": "Information about the binding, format, volume numbers, part of a set, publisher, distributor, etc. which might be enclosed in parenthesis",
+            "type": "string"
```

Comment:
If `qualifier` is always going to be enclosed in parenthesis then would it be better to have a pattern with regex?

Reply:
I just realized from the Jira description, parenthesis may not be always required. This looks ok.